### PR TITLE
Change the layout of MainActivity.

### PR DESCRIPTION
**Pull request by Alexinst**
Changes as followed:
- Use a ScrollView as the outter layout, and a ConstraintLayout as the inner layout. 
- Enlarge the font size.
- Enlarge the button size.

It might be better looking.

### CODE REVIEW

File: app/src/main/java/io/github/trojan_gfw/igniter/MainActivity.java [hunk 1]
```diff
@@ -226,6 +226,8 @@ protected void onResume() {
             updateViews(serviceInstance.getState());
             clashSwitch.setChecked(serviceInstance.enable_clash);
         }
+//        ContactsContract.CommonDataKinds.Phone
```
Comment (wongsyrone) [inline app/src/main/java/io/github/trojan_gfw/igniter/MainActivity.java]: I think we should drop the comment.

Comment (Alexinst) [reply app/src/main/java/io/github/trojan_gfw/igniter/MainActivity.java]: Sorry, I didn't check the changes of "MainActivity.java". Actually, I pulled files from upstream and only modified "activity_main.xml".

**Comment by oasiscifr** [comment]
@Alexinst plz use git rebase to avoid redundant commits begin with "Merge ". You can check out a new branch in your local repo to get some practice before working on this branch.

**Comment by Alexinst** [comment]
@oasiscifr Should I colse this PR and renew another one?

**Comment by oasiscifr** [comment]
@Alexinst Just force push to update.

**Comment by Alexinst** [comment]
@oasiscifr Emmmm, I'm confused now... Maybe, talking in Chinese can help.

**Comment by wongsyrone** [comment]
Our developers can take the responsibility to rebase, please eliminate these useless commits next time.

**Comment by Alexinst** [comment]
I'm sorry for my mistakes.

**Comment by wongsyrone** [comment]
Don't have to say sorry, just practice more.

**Comment by Alexinst** [comment]
Thanks for your suggestion. I will.

**Comment by GreaterFire** [comment]
Manually squashed and merged (111504e5897faa1024f12687a4295e3a72bb64bb). Thanks!

**Pull request closed without merging**